### ISSUE TITLE
Suggest add a README

### ISSUE THREAD

**Issue by DavidHaslam** [teusbenschop/ndebele#50]
Please add a `README.md` file to the repository.

Expand about the project description, copyright status and sponsorship, etc.

Describe how the data structure is designed to be compatible with Bibledit Cloud.

Give some indication of project status stages and timescales, etc.

**Comment by teusbenschop** [comment]
Yes, that's OK. I like the suggestion. Please feel free to add one, and I can merge a pull request. (Whether a subsequent send/receive of Bibledit would leave this file or erase it, we'll see, I hope it will leave the file, but it would need to be tried to be sure.)

**Comment by DavidHaslam** [comment]
In view of your response to my suggestion in Bibledit cloud, I will refrain from adding one.
_Feel free to try it on a fresh repo to see how Bibledit copes._
